### PR TITLE
feat: apply dynamic imports for all components

### PR DESCRIPTION
**Pull request by sniperadmin**
This should enhance the Lighthouse tests

Results:
[DESKTOP VERSION]
image

[MOBILE VERSION]
image

### CODE REVIEW

**Comment by sniperadmin** [comment]
@ivelin now it needs npm to be installed,
I am just confused as we have `yarn`
but anyways, I think this step could make some changes

File: package.json [hunk 1]
```diff
@@ -33,7 +34,8 @@
     "vue-router": "^3.2.0",
     "vue-worker": "^1.2.1",
     "vuetify": "^2.2.29",
-    "vuex": "^3.4.0"
+    "vuex": "^3.4.0",
+    "webpack-bundle-analyzer": "^4.4.0"
```
Comment (vickywane) [inline package.json:38]: You installed new dependencies here. 

Were new lock files generated and have you tried deleting the existing ones alongside your `node_modules`  then running a fresh installation with `npm install` ?

**Review by vickywane — COMMENTED** [review]
Your PR looks great. 
I think you should delete the lock files and `node_modules` then run fresh installations.

If package managers are a source of problems with the CI builds, then other contributors should be aware of the selected package manager.

**Comment by vickywane** [comment]
Netlify checks seem to be breaking, have you had a look @sniperadmin ?

**Comment by sniperadmin** [comment]
looks natural, coz npm was not able to get installed

**Comment by ivelin** [comment]
@sniperadmin CI checks still fail. Converting this PR to draft until all checks are green.

**Review by ivelin — CHANGES_REQUESTED** [review]
There are a number of issues

1. The theme does not respond to the OS change to light/dark as it used to before #445
2. I don’t think we need a light/dark switch at the top menu. The app should respond to the OS settings. If we want to offer an explicit switch, we can put it in settings somewhere, but definitely not as a top navbar button.
3. The timeline view is broken. Nothing renders in the timeline page.

I think going forward we will need a lot more thorough test coverage for UI PRs to avoid regression.

**Comment by ivelin** [comment]
@sniperadmin is this PR still active?

**Comment by ivelin** [comment]
Closing due to inactivity for over 30 days.

**Pull request closed without merging**